packages/verification-vue-component/src/components/Verification/items/VerificationItem.vue

```diff
@@ -31,6 +31,10 @@ export default {
         verificationItem: {
             type: Object,
             required: true
+        },
+        certifactionApiUrl: {
+            type: String,
+            required: false
```

Comment:
Maybe set it to required, as it's required ;-)

Reply:
done